pkg/controller/cluster/main-controller.go

```diff
@@ -435,6 +437,19 @@ func (c *Controller) syncHandler(key string) error {
 		}
 	}
 
+	// List all MinIO instances in this namespace.
+	li, err := c.tenantsLister.Tenants(mi.Namespace).List(labels.NewSelector())
+	if err != nil {
+		return err
+	}
+	// Only 1 minio tenant per namespace allowed.
+	if len(li) > 1 {
```

Comment:
Should this be >=

Reply:
this is correct, `>=` would deny any tenant in any namespace